pom.xml

```diff
@@ -330,6 +330,27 @@ under the License.
 				</executions>
 			</plugin>
 
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>2.17</version>
+				<executions>
+					<execution>
+						<id>test</id>
+						<phase>test</phase>
+						<configuration>
+							<configLocation>style/checkstyle.xml</configLocation>
```

Comment:
Eh ok ... I should have read a bit further! :-) I think we should rename the folder to "checkstyle" then.

As for the license header check - we already have the `apache-rat-plugin` which also does some of this. Could seem that it's less complete though. We might want to sync them up to avoid double work.